README.md

```diff
@@ -9,68 +9,37 @@
-| MARKDOWN_FLAVOUR     | Input file format flavour. The supported flavors are  `markdown`, `markdown_strict`, `markdown_phpextra`, `markdown_github`, `markdown_mmd`, `commonmark`| markdown_github   |
-| OAUTH_TOKEN          | Github Personal Token. generate it by following this [Guide](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)  | NONE *(Not required If using ssh) |
-| DEBUG                | Enable detailed logging                           | **false** |
-| HEROKU_APP_NAME      | Name of the heroku app. Your docs would be deployed at <HEROKU_APP_NAME>.herokuapp.com | NONE |
-| HEROKU_API_KEY       | You can find the key in the [Account](https://dashboard.heroku.com/account) page (in the **API Key** section) on your heroku dashboard | NONE *(Not required if not deploying to Heroku) |
-| AUTOAPI_PYTHON       | If this is **true**, Yaydoc will crawl your repository and try to extract API documentation for all python packages | **false** |
-| AUTOAPI_PYTHON_PATH  | Relative path from the root of the repository, where autoapi will search for Python packages | By default The entire repository is searched |
-| AUTOAPI_JAVA         | If this is **true**, Yaydoc will crawl your repository and try to extract API documentation for all java packages | **false** |
-| AUTOAPI_JAVA_PATH    | Relative path from the root of the repository, where autoapi will search for Java packages | By default The entire repository is searched |
-| SUBPROJECT_URLS      | Comma seperated URLs of all repositories which should be included when building documentation | NONE |
-| SUBPROJECT_DOCPATHS  | Comma seperated `DOCPATHs` for all subprojects     | By default It is assumed `DOCPATH` is `docs` |
-
-```
-   * : The following environment variables must be specified for yaydoc to work.
-```
-
-**With a configuration file**
-
-You could also use a configuration file instead of setting up environment variables.
-Yaydoc automatically reads from a file named *.yaydoc.yml* if present in the root of the repository.
+Yaydoc will automatically read from a file named *.yaydoc.yml* from the root of the repository.
 
 - Specifying Metadata
 
 ```yaml
 metadata:
-  author: FOSSASIA
-  projectname: Yaydoc
-  version: development
-  debug: true
-  subproject:
-    - url: <URL of Subproject 1>
-      docpath: doc
+  author: FOSSASIA       # Author of the project, default: Github uername or organization
+  projectname: Yaydoc    # Name of the Project, default: Name of the repository
+  version: development   # Version of the Project, default: UTC date of latest deployment
+  debug: true            # Enables detailed logging, default: false 
+  subproject:            # Other projects which should be included when building the documentation
+    - url: <URL of Subproject 1>       # URL of the repository
+      docpath: doc                     # Path of the documentation of the subproject, default: docs
     - url: <URL of subproject 2>
```

Comment:
That was when environment variables were directly set bypassing the config.py. That's no longer the case.